### ISSUE TITLE
InvalidArgumentError

### ISSUE THREAD

**Issue by chenweiqian** [nilboy/tensorflow-yolo#15]
Caused by op u'while_1/Pad_1', defined at:
  File "tools/train.py", line 22, in <module>
    solver = eval(solver_params['name'])(dataset, net, common_params, solver_params)
  File "./yolo/solver/yolo_solver.py", line 32, in __init__
    self.construct_graph()
  File "./yolo/solver/yolo_solver.py", line 62, in construct_graph
    self.total_loss, self.nilboy = self.net.loss(self.predicts, self.labels, self.objects_num)
  File "./yolo/net/yolo_tiny_net.py", line 295, in loss
    tuple_results = tf.while_loop(self.cond1, self.body1, [tf.constant(0), object_num, [class_loss, object_loss, noobject_loss, coord_loss], predict, label, nilboy])
  File "/home/wahaha/anaconda2/envs/tensorflow/lib/python2.7/site-packages/tensorflow/python/ops/control_flow_ops.py", line 2626, in while_loop
    result = context.BuildLoop(cond, body, loop_vars, shape_invariants)
  File "/home/wahaha/anaconda2/envs/tensorflow/lib/python2.7/site-packages/tensorflow/python/ops/control_flow_ops.py", line 2459, in BuildLoop
    pred, body, original_loop_vars, loop_vars, shape_invariants)
  File "/home/wahaha/anaconda2/envs/tensorflow/lib/python2.7/site-packages/tensorflow/python/ops/control_flow_ops.py", line 2409, in _BuildLoop
    body_result = body(*packed_vars_for_body)
  File "./yolo/net/yolo_tiny_net.py", line 187, in body1
    response = tf.pad(response, temp, "CONSTANT")
  File "/home/wahaha/anaconda2/envs/tensorflow/lib/python2.7/site-packages/tensorflow/python/ops/array_ops.py", line 1646, in pad
    return gen_array_ops._pad(tensor, paddings, name=name)
  File "/home/wahaha/anaconda2/envs/tensorflow/lib/python2.7/site-packages/tensorflow/python/ops/gen_array_ops.py", line 2083, in _pad
    name=name)
  File "/home/wahaha/anaconda2/envs/tensorflow/lib/python2.7/site-packages/tensorflow/python/framework/op_def_library.py", line 763, in apply_op
    op_def=op_def)
  File "/home/wahaha/anaconda2/envs/tensorflow/lib/python2.7/site-packages/tensorflow/python/framework/ops.py", line 2395, in create_op
    original_op=self._default_original_op, op_def=op_def)
  File "/home/wahaha/anaconda2/envs/tensorflow/lib/python2.7/site-packages/tensorflow/python/framework/ops.py", line 1264, in __init__
    self._traceback = _extract_stack()

InvalidArgumentError (see above for traceback): Paddings must be non-negative: 9 -3
	 [[Node: while_1/Pad_1 = Pad[T=DT_FLOAT, Tpaddings=DT_INT32, _device="/job:localhost/replica:0/task:0/gpu:0"](while_1/ones_1, while_1/Reshape_2)]]
	 [[Node: gradients/local1/add_grad/tuple/control_dependency_1/_191 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/cpu:0", send_device="/job:localhost/replica:0/task:0/gpu:0", send_device_incarnation=1, tensor_name="edge_11242_gradients/local1/add_grad/tuple/control_dependency_1", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/cpu:0"]()]]

What's the problem? @nilboy
